### DISCUSSION TITLE
[FEATURE] Add REST API data connector / data source

### DISCUSSION THREAD

**Discussion by AntonBiryukovUofC** [great-expectations/great_expectations#4179] [Request For Comment]
**Is your feature request related to a problem? Please describe.**

Often, tabular data can be pulled in a JSON format / CSV string from an external REST API. It would be nice to have a way to request a batch of data via query parameters, similar to what is already done for SQL data connectors.

**Describe the solution you'd like**

Data connector that uses pandas execution engine and queries an endpoint with header/body/query parameters attached.

**Describe alternatives you've considered**

RuntimeDataConnector with python wrapper code to get the data and then feed to `great_expectations`, inline.

**Comment by talagluck** [comment]
Thanks for submitting, @AntonBiryukovUofC ! We will review internally and respond over the next week or so.

**Comment by talagluck** [comment]
Hi @AntonBiryukovUofC - following up. This sounds like an awesome feature, but is not something that we will be able to prioritize for our engineering team to build in the immediate future. That said, we are able to offer guidance and review if you would like to contribute this. Would that be of interest to you?
